Documentation/operations/upgrade.rst

```diff
@@ -341,6 +341,13 @@ communicating via the proxy must reconnect to re-establish connections.
   (previously this was a RHEL8.6-compatible kernel).
 * The previously deprecated ``FromRequires`` and ``ToRequires`` fields of the `CiliumNetworkPolicy` and `CiliumClusterwideNetworkPolicy` CRDs have been removed.
 * This release introduces enabling packet layer path MTU discovery by default on CNI Pod endpoints, this is controlled via the ``enable-endpoint-packet-layer-pmtud`` flag.
+* We now set ``clustermesh.apiserver.tls.authMode`` by default to ``migration`` as
+  a first step to transition to ``cluster`` in a future release. If you are using
```

Comment:
non-blocking nit: The following would improve readability slightly (avoid "we" wording for simplicity). Consider applying this if you have other changes to make in addition to this PR. 
```suggestion
* The ``clustermesh.apiserver.tls.authMode`` option is set to ``migration`` by default as
  a first step to transition to ``cluster`` in a future release. If you are using
```